src/app/example_configs/config.yaml

```diff
@@ -116,7 +116,7 @@ metrics:
     port: 2339
     url: http://localhost:3000
 namespaces:
-  local: default
+  local: parent
```

Comment:
This shouldn't be updated in version control

Reply:
yeah, that should not have made it in